### PR TITLE
[MSG] fixes payment request component collapsing

### PR DESCRIPTION
**Pull request by l2succes**
Fixes: https://github.com/artsy/emission/issues/764

And minor UI fixes to match spec

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/Components/Inbox/Conversations/Message.tsx [hunk 1]
```diff
@@ -69,13 +64,7 @@ const Seperator = styled(DottedLine)`
...
 `
 
-const ArtworkPreviewContainer = styled.View`margin-bottom: 10;`
-
-const ImagePreviewContainer = styled.View`margin-bottom: 10;`
-
-const PDFPreviewContainer = styled.View`margin-bottom: 10;`
-
-const InvoicePreviewContainer = styled.View`margin-bottom: 10;`
+const PreviewContainer = styled.View`margin-bottom: 10;`
```
Comment (sarahscott) [inline src/lib/Components/Inbox/Conversations/Message.tsx]: 😄  nice

File: src/lib/Components/Inbox/Conversations/Preview/InvoicePreview.tsx [hunk 2]
```diff
@@ -1,26 +1,23 @@
...
 import fonts from "../../../../../data/fonts"
-
-import SerifText from "../../../Text/Serif"
-
 import InvertedButton from "../../../Buttons/InvertedButton"
 
 const Container = styled.View`
   border-width: 1;
   border-color: ${colors["gray-regular"]};
-  flex: 1;
```
Comment (sarahscott) [inline src/lib/Components/Inbox/Conversations/Preview/InvoicePreview.tsx]: Was this the culprit?

**Comment by sarahscott** [comment]
LGTM ⚡️

File: src/lib/Components/Inbox/Conversations/Preview/InvoicePreview.tsx (hunk 2 above)
Comment (l2succes) [reply src/lib/Components/Inbox/Conversations/Preview/InvoicePreview.tsx]: Yup!

**Pull request merged**